### PR TITLE
Cart functionality

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Cart Functionality

- Added AddToCart functionality
- Added RemoveFromCart functionality
- Added  QuantityHandler functionality
- Added Dynamic price details

### CODE REVIEW

File: src/components/PriceCard/PriceCard.jsx [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+
+      return acc;
+    },
+    {
+      price: 0,
+      discount: 0,
+    }
+  );
+  const totalAmount = totalPrice.price - totalPrice.discount + 500;
+  console.log(totalAmount);
```
Comment (devdeadviz) [inline src/components/PriceCard/PriceCard.jsx:20]: remove console statements.

**Review by devdeadviz — APPROVED** [review]
The code is clean and your file structure is also good, everything is well structured. Good Work!

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import "./navbar.css";
-
 import { Link } from "react-router-dom";
-
+import { useCart } from "../../contexts/cart-context";
```
Comment (sharmayatendra) [inline src/components/NavBar/NavBar.jsx:4]: you can create a index file & import from there

File: src/components/PriceCard/PriceCard.jsx (hunk 1 above)
Comment (sharmayatendra) [inline src/components/PriceCard/PriceCard.jsx:20]: remove console.log

**Review by sharmayatendra — APPROVED** [review]
overall it is good.
folder structure is also good.
you used good variable names.

just added some suggestions can look into it

**Pull request merged**